bundler.d/bmc.rb

```diff
@@ -1,4 +1,7 @@
 group :bmc do
   gem 'rubyipmi', '>= 0.10.0'
   gem 'redfish_client', '>= 0.6.0'
+  # Changed from a default gem to a bundled gem in Ruby 3.4
+  # See: https://stdgems.org/new-in/3.4/
```

Comment:
Perhaps reflect that it was really needed for rubyipmi instead of our codebase?
```suggestion
  # https://github.com/logicminds/rubyipmi/pull/60
```

Reply:
I see you marked it as resolved, but haven't applied it. Does that mean you don't think we should or missed it in your update?

Reply:
Sorry. I have changed it like this and thought its even more clear why we did this change:

```
  # observer is a transitive dependency of rubyipmi
  # Changed from a default gem to a bundled gem in Ruby 3.4
  # See: https://stdgems.org/new-in/3.4/
```

See: https://github.com/theforeman/smart-proxy/pull/914/files#diff-05c54079449e49b795cfbdb6b546fa1e4ca9c3388cccbb44a9890928eb671379R4

Reply:
Ah yes, but my point of adding the link to the PR was to indicate it's a workaround and it really doesn't belong here.

Reply:
And I just noticed it should have been https://github.com/logicminds/rubyipmi/pull/61.

Reply:
Make sense. I have updated the comment again. Hope this solves it now :)